app/assets/stylesheets/blacklight.scss

```diff
@@ -37,3 +37,30 @@
   @extend .col-md-6;
   @extend .col-sm-12;
 }
+
+.documents-list {
+  .document {
+    position: relative;
+    padding-left: 140px;
+  }
+
+  .index_title {
+    font-size: $font-size-large;
+  }
+
+  .document-counter {
+    display: none;
+  }
+
+  .document-thumbnail {
```

Comment:
In doing my mockup, I couldn't figure out how to do it without changing the markup. Though the end result of your approach looks good to me.